docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -78,6 +78,7 @@ Key Features and Updates
   * FEAT-#4147: Add partial compatibility with Python 3.6 and pandas 1.1 (#4301)
   * FEAT-#4569: Add error message when `read_` function defaults to pandas (#4647)
   * FEAT-#4725: Make index and columns lazy in Modin DataFrame (#4726)
+  * FEAT-#4202: Allow dask past 2022.2.0 (#4769)
```

Comment:
I think we should move this under `Dependencies` clause.

Reply:
Done.